banyand/observability/service.go

```diff
@@ -109,7 +110,7 @@ func (p *metricService) PreRun(ctx context.Context) error {
 			if err != nil {
 				p.l.Warn().Err(err).Msg("Failed to create native observability group")
 			}
-			providers = append(providers, newNativeMeterProvider(SystemScope))
+			providers = append(providers, newNativeMeterProvider(SystemScope, p.metadata))
```

Comment:
You don't need to pass a global variable like SystemScope through a parameter; it can be referred to anywhere.